roles/frameworks/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for frameworks
+frameworks_dcos_cli_image: capgemini/dcos-cli
+frameworks_zk_master_peers: "zk://{{ zookeeper_peers_nodes }}/mesos"
+frameworks_mesos_master_url: "http://{{ ansible_ssh_host }}:{{ mesos_master_port }}"
+frameworks_marathon_url: "http://{{ ansible_ssh_host }}:{{ marathon_port }}"
+
+frameworks_cassandra_enabled: true
```

Comment:
are we going to enable this by default. all frameworks except marathon should be disabled by default i think